spec/features/browsing_item_spec.rb

```diff
@@ -31,7 +31,7 @@
 
   context 'when an entry has a bib. ID and ARK for a resource published in Figgy' do
     before do
-      visit 'catalog/3395923'
+      visit 'catalog/9970446223506421'
```

Comment:
Question: Why is this not 9933959233506421?

Reply:
id 9933959233506421 was not in the fixtures I had in current_json under alma. The id I'm using is from EAL, a more complicated example with 2 viewers that tests UV ark and thumbnail.